### PR TITLE
Updating dependencies - Fixing Channels 

### PR DESCRIPTION
**Pull request by nachocano**
## Proposed Changes

  * Updated `knative.dev/pkg`, `knative.dev/eventing`, `knative.dev/serving`
  * pkg needed some newer `k8s.io/client-go`, which we were overriding to 1.12. I ended up increasing all `k8s.io` deps to `kubernetes-1.15.3`, to match eventing, serving, and pkg.
  * With the newer `k8s.io`, also needed to update controller-runtime (to 0.2.2). I had to do a changes in testing code, and some other client calls, due to this change.
  * Moving message_dispatcher and message_receiver to eventing-contrib in order not to break the NATSS nor the Kafka channels whenever we updated the dep to eventing. There are TODOs in the code with links to issues of what should be removed once we start using event_receiver and event_dispatcher from eventing.
   * Few nits (couchdb controller needed configmaps permissions, added URL to printed columns, code-gen for channels, etc, etc.)

NOTE for the Reviewer(s):
  * Don't be scared about the amount of files changed. They are mostly vendored :)
  * Applied everything (githubsource, awssqssource, camel, couchdbsource, kafkasource, kafkachannel, and natsschannel), and all controllers started.
  * Tested the kafka and natss channels data planes and I was able to see events flowing.
  * This will have to be done for the 0.10 cut, so let's do it soon in case there are any issues...

**Release Note**

```release-note
NONE
```

### CODE REVIEW

File: pkg/channel/message_dispatcher_test.go [hunk 1]
```diff
@@ -0,0 +1,432 @@
+/*
+Copyright 2018 The Knative Authors
```
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_dispatcher_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_receiver_test.go [hunk 2]
```diff
@@ -0,0 +1,164 @@
+/*
+Copyright 2018 The Knative Authors
```
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_receiver_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_dispatcher_test.go (hunk 1 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_dispatcher_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_receiver_test.go (hunk 2 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_receiver_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_dispatcher_test.go (hunk 1 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_dispatcher_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_receiver_test.go (hunk 2 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_receiver_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_dispatcher_test.go (hunk 1 above)
Comment (nachocano) [reply pkg/channel/message_dispatcher_test.go]: old file from eventing... discarding

**Comment by matzew** [comment]
:eyes:  and testing (especially kafka) 

/cc @lberk

**Comment by matzew** [comment]
The KafkaSource works fine

**Comment by matzew** [comment]
In the channel, I am currently getting this error:

```
{"level":"info","ts":"2019-09-30T08:29:32.483Z","logger":"controller","caller":"channel_dispatcher/main.go:72","msg":"Kafka broker configuration - bootstrapServers: [my-cluster-kafka-bootstrap.kafka.:9092]"}
{"level":"fatal","ts":"2019-09-30T08:29:33.243Z","logger":"controller","caller":"channel_dispatcher/main.go:82","msg":"Unable to create kafka dispatcher","error":"unable to create kafka client: kafka: client has run out of available brokers to talk to (Is your cluster reachable?)","stacktrace":"main.main\n\t/home/matzew/go/src/knative.dev/eventing-contrib/kafka/channel/cmd/channel_dispatcher/main.go:82\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:200"}
```

**Comment by matzew** [comment]
I got the error sorted (typo) on the URL.

Now, I could create a channel, and saw the native kafka topic for the channel.  Posting messages to the channel, but no delivery of messages to the subscribed `ksvc`.

In the dispatcher pod, I get:

```
{"level":"info","ts":"2019-09-30T08:32:33.931Z","logger":"controller.kafka-ch-dispatcher","caller":"controller/controller.go:351","msg":"Reconcile succeeded. Time taken: 104.527µs.","controller/impl":"pkg","knative.dev/controller":"kafka-ch-dispatcher","knative.dev/traceid":"cb96b853-4d5a-4156-843c-6b4132dec4bf","knative.dev/key":"default/ctestchannel"}
{"level":"info","ts":"2019-09-30T08:32:33.931Z","logger":"controller.kafka-ch-dispatcher.event-broadcaster","caller":"record/event.go:258","msg":"Event(v1.ObjectReference{Kind:\"KafkaChannel\", Namespace:\"default\", Name:\"ctestchannel\", UID:\"41f7c4d9-7d99-40fa-af0c-8b8432ad5be4\", APIVersion:\"messaging.knative.dev/v1alpha1\", ResourceVersion:\"3672\", FieldPath:\"\"}): type: 'Normal' reason: 'ChannelReconciled' KafkaChannel reconciled","controller/impl":"pkg","knative.dev/controller":"kafka-ch-dispatcher"}
E0930 08:32:33.933732       1 event.go:240] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"ctestchannel.15c92a355c04171d", GenerateName:"", Namespace:"default", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:v1.Time{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, DeletionTimestamp:(*v1.Time)(nil), DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Initializers:(*v1.Initializers)(nil), Finalizers:[]string(nil), ClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"KafkaChannel", Namespace:"default", Name:"ctestchannel", UID:"41f7c4d9-7d99-40fa-af0c-8b8432ad5be4", APIVersion:"messaging.knative.dev/v1alpha1", ResourceVersion:"3672", FieldPath:""}, Reason:"ChannelReconciled", Message:"KafkaChannel reconciled", Source:v1.EventSource{Component:"kafka-ch-dispatcher", Host:""}, FirstTimestamp:v1.Time{Time:time.Time{wall:0xbf5c8d486e890d1d, ext:26735178057, loc:(*time.Location)(0x29e7ae0)}}, LastTimestamp:v1.Time{Time:time.Time{wall:0xbf5c8d48778016a0, ext:26885585612, loc:(*time.Location)(0x29e7ae0)}}, Count:2, Type:"Normal", EventTime:v1.MicroTime{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'events "ctestchannel.15c92a355c04171d" is forbidden: User "system:serviceaccount:knative-eventing:kafka-ch-dispatcher" cannot patch resource "events" in API group "" in the namespace "default"' (will not retry!)
```

File: kafka/channel/pkg/dispatcher/dispatcher.go [hunk 3]
```diff
@@ -29,8 +29,9 @@ import (
 	"go.uber.org/zap"
 
 	"knative.dev/eventing-contrib/kafka/common/pkg/kafka"
+	channels "knative.dev/eventing-contrib/pkg/channel"
```
Comment (matzew) [inline kafka/channel/pkg/dispatcher/dispatcher.go]: contribchannels, instead of channels ?

**Comment by matzew** [comment]
@nachocano I had to add this, and it all worked:

https://github.com/nachocano/eventing-contrib/pull/7

( I am on minikube 1.4.0 and k8s: 1.15.4)

File: pkg/channel/message_dispatcher_test.go (hunk 1 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_dispatcher_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_receiver_test.go (hunk 2 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_receiver_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

**Comment by nachocano** [comment]
> @nachocano I had to add this, and it all worked:
> 
> [nachocano#7](https://github.com/nachocano/eventing-contrib/pull/7)
> 
> ( I am on minikube 1.4.0 and k8s: 1.15.4)

Awesome, thanks @matzew for testing it!!!
Let me do the alias change, and see why integrations are testing

**Comment by nachocano** [comment]
@matzew after I merged your PR, it seems that I need your consent... Can you comment `@googlebot I consent`

File: pkg/channel/message_dispatcher_test.go (hunk 1 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_dispatcher_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_receiver_test.go (hunk 2 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_receiver_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_dispatcher_test.go (hunk 1 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_dispatcher_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_receiver_test.go (hunk 2 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_receiver_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: kafka/channel/pkg/dispatcher/dispatcher.go (hunk 3 above)
Comment (nachocano) [reply kafka/channel/pkg/dispatcher/dispatcher.go]: done!

**Comment by nachocano** [comment]
@sayanh @Abd4llA can you guys test that the NATSS channel still works with this change, whenever you have few cycles?... I tested in my cluster and I'm able to receive events... But just wanted to double check... I'm not entirely sure why the integration test failed...

**Comment by matzew** [comment]
@nachocano I noticed smth like this before:  https://github.com/knative/eventing-contrib/issues/627

**Comment by nachocano** [comment]
> @nachocano I noticed smth like this before: #627

hmm, yeah... but is weird that in master this one is not failing... might have messed up something... will keep on debugging

**Comment by matzew** [comment]
@googlebot I consent

**Comment by chizhg** [comment]
Did we change something in `eventing-contrib` or `eventing`?
The tests started failing consistently from this morning, see https://testgrid.knative.dev/eventing-contrib#continuous

**Comment by nachocano** [comment]
> Did we change something in `eventing-contrib` or `eventing`?
> The tests started failing consistently from this morning, see https://testgrid.knative.dev/eventing-contrib#continuous

Yes, this is a big change that hasn't been merged yet... It seems that something got broken...
I've increased many dependencies versions... You can see the changes here...
Kafka is working, but there seems to be a problem with NATSS... Will debug later today to see what is going on...

**Comment by nachocano** [comment]
It is not broken in master right? It's just this PR... Or is broken in master?

**Comment by vaikas-google** [comment]
It's broken in the master, I'm thinking this might have had something to
do with it, but only based on the timelines, no smoking gun:
https://github.com/knative/eventing/pull/1972

**Comment by chizhg** [comment]
> It is not broken in master right? It's just this PR... Or is broken in master?

`eventing-contrib` is installing the HEAD of `eventing` for its e2e testing, so it can be some changes in `eventing` that break it.

**Comment by nachocano** [comment]
> `eventing-contrib` is installing the HEAD of `eventing` for its e2e testing, so it can also be some changes in `eventing` that break `eventing-contrib` master.

Yes, a change that I did sometime back on using event_dispatcher and event_receiver should have broken contrib then... But that change was done 1 week back... Wondering why it didn't break before... This was the change... https://github.com/knative/eventing/pull/1933

**Comment by nachocano** [comment]
@chizhg 
ran in my cluster `go test -v  -race -count=1 -tags=e2e -timeout=20m -parallel=12 ./test/e2e -channels=NatssChannel`, and all tests pass... especially the two that are failing in PROW:

```
--- PASS: TestEventTransformationForTrigger (1.14s)
    --- PASS: TestEventTransformationForTrigger/TestEventTransformationForTrigger-NatssChannel (60.10s)
...
--- PASS: TestBrokerChannelFlow (1.13s)
    --- PASS: TestBrokerChannelFlow/TestBrokerChannelFlow-NatssChannel (71.72s)
```

Any idea of what can be happening... Have the HEAD of eventing and eventing-contrib (plus my changes)

File: pkg/channel/message_dispatcher_test.go (hunk 1 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_dispatcher_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_receiver_test.go (hunk 2 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_receiver_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

**Comment by nachocano** [comment]
> > `eventing-contrib` is installing the HEAD of `eventing` for its e2e testing, so it can also be some changes in `eventing` that break `eventing-contrib` master.
> 
> Yes, a change that I did sometime back on using event_dispatcher and event_receiver should have broken contrib then... But that change was done 1 week back... Wondering why it didn't break before... This was the change... [knative/eventing#1933](https://github.com/knative/eventing/pull/1933)

I think it didn't fail because the channels code is using a vendored version of eventing and not my latest changes...

File: pkg/channel/message_dispatcher_test.go (hunk 1 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_dispatcher_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_receiver_test.go (hunk 2 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_receiver_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: test/e2e-tests.sh [hunk 4]
```diff
@@ -73,6 +73,13 @@ function knative_setup() {
     popd
   fi
   wait_until_pods_running knative-eventing || fail_test "Knative Eventing did not come up"
+
+  # TODO install head if !is_release_branch
```
Comment (chizhg) [inline test/e2e-tests.sh]: Just curious, do we have plan to move `monitoring` out of `serving`?

File: pkg/channel/message_dispatcher_test.go (hunk 1 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_dispatcher_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_receiver_test.go (hunk 2 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_receiver_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: test/e2e-tests.sh (hunk 4 above)
Comment (nachocano) [reply test/e2e-tests.sh]: I think so, but that may take sometime.

File: pkg/channel/message_dispatcher_test.go (hunk 1 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_dispatcher_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_receiver_test.go (hunk 2 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_receiver_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: Gopkg.toml [hunk 5]
```diff
@@ -89,7 +89,8 @@ required = [
 
 [[constraint]]
   name = "knative.dev/eventing"
-  branch = "master"
+  source = "github.com/nachocano/eventing"
+  branch = "tracing"
```
Comment (nachocano) [inline Gopkg.toml]: /hold 
just to see if this makes it work

File: pkg/channel/logger_test.go [hunk 6]
```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2018 The Knative Authors
```
Comment (mattmoor-sockpuppet) [inline pkg/channel/logger_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_dispatcher_test.go (hunk 1 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_dispatcher_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_receiver_test.go [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
```
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_receiver_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/references_test.go [hunk 8]
```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2018 The Knative Authors
```
Comment (mattmoor-sockpuppet) [inline pkg/channel/references_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/logger_test.go (hunk 6 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/logger_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_dispatcher_test.go (hunk 1 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_dispatcher_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_receiver_test.go (hunk 7 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_receiver_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/references_test.go (hunk 8 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/references_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/logger_test.go (hunk 6 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/logger_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_dispatcher_test.go (hunk 1 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_dispatcher_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_receiver_test.go (hunk 7 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_receiver_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/references_test.go (hunk 8 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/references_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

**Comment by nachocano** [comment]
There is a problem with the patching that we are doing to opencensus:

`vendor/knative.dev/eventing/pkg/tracing/traceparent.go:83:6: span.SetSpanID undefined (type *trace.Span has no field or method SetSpanID)`

File: pkg/channel/message_dispatcher_test.go (hunk 1 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_dispatcher_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_receiver_test.go (hunk 7 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_receiver_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_dispatcher_test.go (hunk 1 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_dispatcher_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_receiver_test.go (hunk 2 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_receiver_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: hack/set-span-id.patch [hunk 9]
```diff
@@ -0,0 +1,16 @@
+diff --git a/vendor/go.opencensus.io/trace/trace.go b/vendor/go.opencensus.io/trace/trace.go
+index 38ead7bf..9e6fe483 100644
+--- a/vendor/go.opencensus.io/trace/trace.go
++++ b/vendor/go.opencensus.io/trace/trace.go
+@@ -261,6 +261,11 @@ func startSpanInternal(name string, hasParent bool, parent SpanContext, remotePa
+ 	return span
+ }
+
```
Comment (mattmoor-sockpuppet) [inline hack/set-span-id.patch]: Remove trailing whitespace:
```suggestion

```

File: pkg/channel/message_dispatcher_test.go (hunk 1 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_dispatcher_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_receiver_test.go (hunk 2 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_receiver_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_dispatcher_test.go (hunk 1 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_dispatcher_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_receiver_test.go (hunk 2 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_receiver_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: hack/set-span-id.patch (hunk 9 above)
Comment (mattmoor-sockpuppet) [inline hack/set-span-id.patch]: Remove trailing whitespace:
```suggestion

```

**Comment by nachocano** [comment]
/hold cancel
doing the patching that eventing does for now

File: pkg/channel/message_dispatcher_test.go (hunk 1 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_dispatcher_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_receiver_test.go (hunk 2 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_receiver_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: hack/set-span-id.patch (hunk 9 above)
Comment (mattmoor-sockpuppet) [inline hack/set-span-id.patch]: Remove trailing whitespace:
```suggestion

```

File: pkg/channel/message_dispatcher_test.go (hunk 1 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_dispatcher_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_receiver_test.go (hunk 2 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_receiver_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: hack/set-span-id.patch (hunk 9 above)
Comment (mattmoor-sockpuppet) [inline hack/set-span-id.patch]: Remove trailing whitespace:
```suggestion

```

File: pkg/channel/message_dispatcher_test.go (hunk 1 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_dispatcher_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_receiver_test.go (hunk 2 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_receiver_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_dispatcher_test.go (hunk 1 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_dispatcher_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: pkg/channel/message_receiver_test.go (hunk 2 above)
Comment (mattmoor-sockpuppet) [inline pkg/channel/message_receiver_test.go]: Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

File: hack/set-span-id.patch (hunk 9 above)
Comment (mattmoor-sockpuppet) [inline hack/set-span-id.patch]: Remove trailing whitespace:
```suggestion

```

**Comment by nachocano** [comment]
/hold cancel

we are skipping the two e2e tests failing for now. Created an issue https://github.com/knative/eventing-contrib/issues/639 to unblock us.... We narrowed it down to a particular PR in eventing, which we will continue to investigate... 

fyi @chizhg @Harwayne

**Pull request merged**